docs/customize/index.md

```diff
@@ -9,7 +9,8 @@ InvenioRDM instance so that it integrates into their organizational environment.
 
 - [Styling](styling.md) - change the look and feel of InvenioRDM to match your institution.
 - [Authentication](authentication.md) - integrate with your institutional authentication system.
-- [DOI registration](dois.md) - mint DOIs with your institution's prefix
-- [Application data](application_data.md) - use custom controlled vocabularies, default users and other institution specific data
+- [DOI registration](dois.md) - mint DOIs with your institution's prefix.
+- [Application data](application_data.md) - use custom controlled vocabularies, default users and other institution specific data.
+- [Landing page](landing_page.md) - customize some sections of the landing page.
```

Comment:
I would specify that it is the landing page of the record, everywhere. It is not clear to me, otherwise, which landing page:
```suggestion
- [Record's landing page](record_landing_page.md) - customize some of the sections of record's landing page.
```